src/backend/distributed/metadata/metadata_utility.c

```diff
@@ -2231,11 +2236,10 @@ UpdatePgDistPartitionAutoConverted(Oid citusTableId, bool autoConverted)
 							   citusTableId)));
 	}
 
-	memset(replace, 0, sizeof(replace));
-
-	values[Anum_pg_dist_partition_autoconverted - 1] = BoolGetDatum(autoConverted);
-	isnull[Anum_pg_dist_partition_autoconverted - 1] = false;
-	replace[Anum_pg_dist_partition_autoconverted - 1] = true;
+	autoconvertedindex = GetAutoConvertedAttrIndexInPgDistPartition(tupleDescriptor);
```

Comment:
nit - let's declare this right away here

```suggestion
	int autoconvertedindex = GetAutoConvertedAttrIndexInPgDistPartition(tupleDescriptor);
```